src/api-routes/persons.js

```diff
@@ -11,6 +11,10 @@ const post = async (req, res) => {
   try {
     const { body: { data: { attributes } } } = req;
 
+    if (new Date(attributes.birthDate) > new Date()) {
```

Comment:
Since we have `moment` installed in the skeleton, I personally recommend using it for all time/date related codes since it's more clear in my opinion: https://momentjs.com/docs/

E.g. You can use `moment#isAfter` in this case.